.github/workflows/build_and_test_workflow.yml

```diff
@@ -152,9 +152,9 @@ jobs:
           npm uninstall -g yarn
           npm i -g yarn@1.22.10
 
-      # image has the latest chrome v99
+      # match version in package.json
       - name: Setup chromedriver
-        run: yarn add --dev chromedriver@99.0.0
+        run: yarn add --dev chromedriver@107.0.3
```

Comment:
@kavilla how do we bump this version in the Docker image?

Reply:
@tmarkley I have #2990 to automate this at runtime.

Reply:
@AMoo-Miki I just noticed that PR this morning, it's great! I've already updated this PR to remove these parts.